pkg/index/index.go

```diff
@@ -422,26 +423,7 @@ func (x *Index) isEmpty() bool {
 }
 
 // reindexMaxProcs is the number of concurrent goroutines that will be used for reindexing.
-var reindexMaxProcs = struct {
-	sync.RWMutex
-	v int
-}{v: 4}
-
-// SetReindexMaxProcs sets the maximum number of concurrent goroutines that are
-// used during reindexing.
-func SetReindexMaxProcs(n int) {
-	reindexMaxProcs.Lock()
-	defer reindexMaxProcs.Unlock()
-	reindexMaxProcs.v = n
-}
-
-// ReindexMaxProcs returns the maximum number of concurrent goroutines that are
-// used during reindexing.
-func ReindexMaxProcs() int {
-	reindexMaxProcs.RLock()
-	defer reindexMaxProcs.RUnlock()
-	return reindexMaxProcs.v
-}
```

Comment:
This seems unrelated?

Reply:
Keep the setter/getter and just change the default of 4 to NumCPU?

Reply:
:heavy_check_mark: